.github/workflows/ci.yml

```diff
@@ -0,0 +1,131 @@
+name: Moodle plugin CI for main
+
+# Controls when the action will run.
+on: [push, pull_request]
+
+jobs:
+  citest:
+    name: CI test
+    runs-on: 'ubuntu-latest'
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: 'postgres'
+          POSTGRES_HOST_AUTH_METHOD: 'trust'
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 3
+        ports:
+          - 5432:5432
+
+      mariadb:
+        image: mariadb
+        env:
+          MYSQL_USER: 'root'
+          MYSQL_ALLOW_EMPTY_PASSWORD: "true"
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 3
+    strategy:
+      fail-fast: false
+      matrix:
+        database: ['pgsql', 'mariadb']
+        moodle-branch: ['MOODLE_39_STABLE']
+        node: ['14.15.1']
+        php: ['7.2', '7.3']
+        include:
+          - {moodle-branch: 'MOODLE_39_STABLE', php: '7.4', node: '14.15.1', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_39_STABLE', php: '7.4', node: '14.15.1', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_310_STABLE', php: '7.4', node: '14.15.1', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_310_STABLE', php: '7.4', node: '14.15.1', database: 'pgsql'} 
+          - {moodle-branch: 'MOODLE_311_STABLE', php: '7.4', node: '14.15.1', database: 'pgsql'}
```

Comment:
wondering why yo decided to change an order of DBs? :)

Reply:
fixed it